custom_dc/custom_data.md

```diff
@@ -0,0 +1,291 @@
+
+#### Input file parameters
+
+`entityType`
+
+: Required: All entities in a given file must be of a specific type. This type should be specified as the value of the <code>entityType</code> field. The importer tries to resolve entities to DCIDs of that type. In most cases, the <code>entityType</code> will be a supported place type; see [Place types](../place_types.md) for a list.
+
+`ignoreColumns`
+
+: Optional: The list of column names to be ignored by the importer, if any.
+
+`provenance`
+
+: Required: The provenance (name) of this input file. Provenances typically map to a dataset from a source. For example, `WorldDevelopmentIndicators` provenance (or dataset) is from the `WorldBank` source.  
+
+You must specify the provenance details under `sources`.`provenances`; this field associates one of the provenances defined there to this file.
+
+
+### Variables
+
+The `variables` section is optional. You can use it to override names and associate additional properties with the statistical variables in the files, using the parameters described below. All parameters are optional.
+
+#### Variable parameters
+
+`name`
+
+: The display name of the variable, which will show up in the site's exploration tools. If not specified, the column name is used as the display name.  
+The name should be concise and precise; that is, the shortest possible name that allow humans to uniquely identify a given variable. The name is used to generate NL embeddings.
+
+`description`
+
+: A long-form description of the variable.
+
+`properties` 
+
+: Additional Data Commons properties associated with this variable. These are Data Commons property entities. See [Representing statistics in Data Commons](../data/blob/master/docs/representing_statistics.md#statisticalvariable) for more details.  
+
+Each property is specified as a key:value pair. Here are some examples:  
+
+```json
+{
+  "populationType": "schema:Person",  
+  "measuredProperty": "age",  
+  "statType": "medianValue",  
+  "gender": "Female" 
+} 
+```
+
+`group`
+
+: You can arrange variables in groups, so that they appear together in the Statistical Variables Explorer and other exploration tools. The group name is used as the heading of the group. For example, in the sample data, the group name `OECD` is used to group together the two variables from the two CSV files:
+
+![group_screenshot](/assets/images/custom_dc/customdc_screenshot5.png){: width="250"}
+
+You can have a multi-level group hierarchy by using `/` as a separator between each group.
+
+`earchDescriptions`
+
+: Formerly `nlSentences`. An array of descriptions to be used for creating more NL embeddings for the variable. This is only needed if the variable `name` is not sufficient for generating embeddings.  
```

Comment:
Given it's a new doc, can we remove `nlSentences` from this doc completely?

Reply:
Yep, good point.